mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -136,25 +136,43 @@ properties:
   - !ruby/object:Api::Type::Time
     name: "createTime"
     description: |
-      Time the Instance was created in UTC.
+      Time when this resource was created.
     output: true
+  - !ruby/object:Api::Type::String
+    name: "idleTimeout"
+    default_from_api: true
```

Comment:
As it states in the description, the fields seems to have a default value. Do we know if it's a fixed default in the API. If so, we probably can use `default_value` instead of `default_from_api`. Also, can we add a test with `0s`. Just want to make that the API will return `0s` in this case, otherwise, it will cause perma-diff in Terraform.

Reply:
I'm going to split this out into a separate PR, because the API cannot test this case at this time.